inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/node/base/ExtractNodeProvider.java

```diff
@@ -60,8 +61,23 @@ public interface ExtractNodeProvider extends NodeProvider {
      */
     default List<FieldInfo> parseStreamFieldInfos(List<StreamField> streamFields, String nodeId) {
         // Filter constant fields
+        return parseStreamFieldInfos(streamFields, nodeId, null);
+    }
+
+    /**
+     * Parse StreamFieldInfos
+     *
+     * @param streamFields The stream fields
+     * @param nodeId The node id
+     * @param dataTypeMappingStrategy The data type mapping operation strategy
+     * @return FieldInfo list
+     */
+    default List<FieldInfo> parseStreamFieldInfos(List<StreamField> streamFields, String nodeId,
```

Comment:
This method is redundant.Just rewrite `parseStreamFieldInfos(List<StreamField> streamFields, String nodeId)`

Reply:
> parseStreamFieldInfos

overloaded method